log/sequencer.go

```diff
@@ -37,7 +43,13 @@ type CurrentRootExpiredFunc func(trillian.SignedLogRoot) bool
 
 // NewSequencer creates a new Sequencer instance for the specified inputs.
 func NewSequencer(hasher merkle.TreeHasher, timeSource util.TimeSource, logStorage storage.LogStorage, km crypto.KeyManager) *Sequencer {
-	return &Sequencer{hasher, timeSource, logStorage, km}
+	return &Sequencer{hasher: hasher, timeSource: timeSource, logStorage: logStorage, keyManager: km}
+}
+
+// SetGuardWindow changes the interval that must elapse between leaves being queued and them
+// being eligible for sequencing. The default is a zero interval.
+func (s *Sequencer) SetGuardWindow(sequencerGuardWindow time.Duration) {
+	s.sequencerGuardWindow = sequencerGuardWindow
```

Comment:
As above. It's safe for now. Will need sorting out when we implement readonly support and related admin things.